src/MAUI/Maui.Samples/AppShell.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Shell
```

Comment:
On MacCatalyst - the categories text are aligned to the left without any margin which doesn't look great. Unsure if this can be fixed.

Reply:
Less problematic in fullscreen mode

Reply:
This is a challenge to address, I had tried something before but it had negative impacts on the other platforms. I'll take another look.